preview-env/clean/github.sh

```diff
@@ -224,7 +224,8 @@ function get_pull_requests_by_states_with_last_100_comments {
       "${state_fields[@]}" \
       --field "query=@$RELATIVE_SCRIPT_PATH/graphql/query_get_pull_requests_by_states_with_last_100_comments.gql" |
       jq -s \
-        '[.[].data.repository.pullRequests.nodes[]]'
+        '[.[].data.repository.pullRequests.nodes[]]' |
+      jq '(.[].comments.nodes[] | select(.body | length > 1000)).body |= "Filtered very long comment"'
```

Comment:
Thx a lot @cmur2 🙇.

Two small suggestions:
- add the new directive to the existing jq command
- truncate the body so that at least some of the content is present
```suggestion
        '[.[].data.repository.pullRequests.nodes[]] |
         (.[].comments.nodes[] | select(.body | length > 1000)).body |= .[0:980] + " ... (truncated)"'
```

Reply:
Thank you for the suggestions, I added them and they work fine as shown in https://github.com/camunda/camunda-optimize/actions/runs/6783837906/job/18475647014